ui/src/components/facets/FacetCard.js

```diff
@@ -95,8 +90,11 @@ class FacetCard extends Component {
         onClick={e => this.onClick(facetValue.name)}
       >
         <Checkbox
-          className={classes.facetValueCheckbox}
-          checked={this.state.selectedValues.includes(facetValue.name)}
+          style={{ width: "24px", height: "24px" }}
```

Comment:
Delete, this are in facetValueCheckbox above

Reply:
Ping

Reply:
Done